charts/jiralert/templates/deployment.yaml

```
@@ -67,7 +67,7 @@ spec:
           {{- end }}
         {{- with .Values.env }}
           env:
-          {{- range $key, $value := .Values.env }}
+          {{- range $key, $value := $.Values.env }}
```

```suggestion
          {{- range $key, $value := . }}
```

would make more sense?